src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -1180,6 +1239,11 @@
 type: command
 short-summary: >
   Remove a managed rule set by rule set group name if rule_group_name is specified. Otherwise, remove all rule set.
+examples:
+  - name: Remove a managed rule set by rule set group name if rule_group_name is specified. Otherwise, remove all rule set.
```

Comment:
without `(autogenerated)` but with `crafted: true`

Reply:
`(autogenerated)` is shown to users.